packages/state-transition/src/cache/epochCache.ts

```diff
@@ -918,12 +918,22 @@ export class EpochCache {
    * balances are sampled to adjust the probability of the next selection (32 per epoch on average). So to invalidate
    * the prediction the effective of one of those 32 samples should change and change the random_byte inequality.
    */
-  getBeaconProposersNextEpoch(): ValidatorIndex[] {
+  async getBeaconProposersNextEpoch(): Promise<ValidatorIndex[]> {
     if (!this.proposersNextEpoch.computed) {
+      if (!this.nextShuffling) {
+        this.nextShuffling = (await this.shufflingCache?.get(this.nextEpoch, this.nextDecisionRoot)) ?? null;
+      }
+      if (!this.nextShuffling) {
+        throw new EpochCacheError({
```

Comment:
I suppose we never reach this and this is just to make the compiler happy, drop some comments for that context

Reply:
yes correct. was removed in refactor